pkg/internal/checkup/executor/executor.go

```diff
@@ -32,6 +32,7 @@ import (
 	"kubevirt.io/client-go/kubecli"
 
 	"github.com/kiagnose/kubevirt-dpdk-checkup/pkg/internal/checkup/console"
+	trex "github.com/kiagnose/kubevirt-dpdk-checkup/pkg/internal/checkup/trex-client"
```

Comment:
Please consider renaming the `trex-client` directory to `trex`.
1. It would make this alias redundant.
2. It would give the `trex` prefix to its objects:
```go
trex.NewClient()
trex.GlobalStats
```

Same for the files under the `trex` directory:
```
trex/types.go
trex/client.go
```

Reply:
ahh my bad. DONE